### PR TITLE
minor ui correction

### PR DESCRIPTION
**Pull request by yakubsubasi**
## What kind of change does this PR introduce?

minor ui correction

## What is changed?

- The ForgetPassword button has been removed from the ForgetPassword page, as there is no need for a redirect.

- Added appbar titles

### CODE REVIEW

File: example/lib/sign_up.dart [hunk 1]
```diff
@@ -10,7 +10,7 @@ class SignUp extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: const Text('auth ui example'),
+        title: const Text('Sign Up'),
```
Comment (dshukertjr) [inline example/lib/sign_up.dart:13]: Could we replace this whole app bar with `appBar('Sign Up')`?

**Review by dshukertjr — COMMENTED** [review]
@yakubsubasi This is great! Thanks for the amazing work! I just had one thing, but other than that, everything looks amazing!

**Review by dshukertjr — APPROVED** [review]
Looks great! Thanks for the amazing work!

**Comment by yakubsubasi** [comment]
> Looks great! Thanks for the amazing work!

You're welcome. I didn't actually do much.
The package has a lot of room for improvement compared to [firebase_ui_auth](https://pub.dev/packages/firebase_ui_auth). I'd like to work on solving localization issues in the repo. (I need to use it in Turkish app). I'm excited to make more meaningful contributions, but as a new dev, I could use some guidance.

**Comment by dshukertjr** [comment]
@yakubsubasi 
That would be amazing! Yes, we haven't been able to spend as much time as we wanted to on this package, so it certainly has so much room for improvements! Any contributions would be greatly appreciated! 

Also, is this your first contribution to Supabase? Have you received Supabase swag for it? If not, since you have contributed to this library, I can send you a swag code! DM me on [Twitter](https://twitter.com/dshukertjr)!

**Comment by yakubsubasi** [comment]
Thanks a lot @dshukertjr. Yes, that was my first. I have sent a message on twitter.

**Pull request merged**